tests/acceptance/ManageSegmentsCest.php

```diff
@@ -46,6 +48,17 @@ public function viewUserRoleSegmentSubscribers(\AcceptanceTester $i) {
     $i->dontSee($wpAdminEmail);
     $i->dontSee($wpAuthorEmail);
     $i->seeNoJSErrors();
+
+    $i->wantTo('Set pagination to 1 user per page and check if pagination is present');
+    $i->click('#show-settings-link');
+    $i->fillField('#mailpoet_subscribers_per_page', '1');
+    $i->click('#screen-options-apply');
+    $i->wait(2); // to avoid flakyness, required to wait a bit
+    $i->waitForElement('.mailpoet-listing-pages-next');
+    $i->waitForText($wpEditorEmail, 20);
+    $i->click('.mailpoet-listing-pages-next');
+    $i->waitForText($wpEditorEmail2, 20);
+    $i->makeScreenshot('ss1');
```

Comment:
I wonder why do we need to take a screenshot as the last step here? Codeception does that automatically for failures. What is the idea behind taking such a screenshot or isn't it just a leftover from testing?

Reply:
Ah, can't believe I left it again. Will pay more attention next time, sorry. Removed it.